examples/daisyworld.jl

```diff
@@ -335,3 +336,65 @@ p2 = plot(agent_df[!, :step], agent_df[!, aggname(adata[3])], ylabel = "temperat
 p3 = plot(model_df[!, :step], model_df[!, :solar_luminosity], ylabel = "L", xlabel = "ticks")
 
 plot(p, p2, p3, layout = (3, 1))
+
+
+# ## Interactive scientific research
+# Julia is an interactive language, and thus everything that you do with Agents.jl can be
+# considered interactive. However, we can do even better by using our interactive application.
+# In this example, we care about exploring some time-dependent forcing of the system, and
+# so far we have been pre-describing it. But with using the interactive application,
+# we can control by ourselves, in real time, how much to change the solar forcing.
+
+# So, let's use [`interactive_abm`](@ref)!
+
+# ```julia
+# using InteractiveChaos, Makie, Random
+# Random.seed!(165)
+# model = daisyworld(; solar_luminosity = 1.0, solar_change = 0.0, scenario = :change)
+# ```
+
+# Thankfully, we have already defined the necessary `adata, mdata` as well as the agent
+# color/shape/size functions, and we can re-use them for the interactive application.
+# Unfortunately, because `InteractiveChaos` uses a different plotting package, Makie.jl, we have
+# to redefine the plotting functions. However, in the near future where AgentsPlots.jl
+# will move to Makie.jl, this will not be necessary.
```

Comment:
No need to be negative about improvements! A suggested change:

Because `InteractiveChaos` uses a different plotting package: Makie.jl, the plotting functions we define here are slightly different to our normal routine.
AgentsPlots.jl is planning to also move to Makie.jl, so something similar to this syntax will be normal in the near future.